packages/7-todo-tdd/src/test/config.ts

```diff
@@ -0,0 +1,6 @@
+import enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+enzyme.configure({ adapter: new Adapter() });
+
+jest.mock('../services/TodoService', () => require('../__mocks__/TodoService'));
```

Comment:
instead of replacing the whole service. u can mock the response for each function as u need it (in the given part)